qiita_db/metadata_template.py

```diff
@@ -1486,6 +1486,15 @@ def extend(self, md_template):
         # adding the fp to the object
         self.add_filepath(fp)
 
+    @property
+    def headers(self):
+        """Identifies the metadata columns present in a template"""
+
+        cols = get_table_cols(self._table_name(self._id))
+        cols.extend(get_table_cols(self._table)[1:])
+
```

Comment:
Can you run this through the `self.translate_cols_dict` so the column names are consistent with the templates instead of the database?